tensorboard/plugins/graph/tf_graph_loader/tf-graph-loader.html

```diff
@@ -203,6 +247,8 @@
       // Generic error catch, for errors that happened outside
       // asynchronous tasks.
       tracker.reportError("Graph visualization failed: " + e, e);
+      // Don't swallow the error.
+      throw e;
```

Comment:
> Or is there a
particular motivation for including it together with these changes?

When we fetch graph then profile, we would like to fail as soon as we fail to fetch the graph. Any reason why this is wide-impact modulo unhandled Promise?

Reply:
I was thinking that any existing consumers of this promise may have been
depending on it resolving rather than rejecting even if the
visualization fails. But on second thought that doesn’t seem likely,
because the promise is only returned as of this commit. So this is fine
with me.